src/qana/RelationGraph.cc

```diff
@@ -878,6 +882,27 @@ void computeMinimumOverlap(Vertex& vtx) {
                 // vertex visitation queue.
                 u->overlap = requiredOverlap;
                 q.enqueue(u);
+                LOGS(_log, LOG_LVL_TRACE, " (updated and queued)");
+            }
+        }
+    }
+}
+
+/// dump graph
+void dumpGraph(std::list<Vertex> const& vertices) {
```

Comment:
Shouldn't this free-standing function be put into the anonymous namespace at the beginning of this compilation unit?

Reply:
I don't see any harm to have it in the `lsst::qserv::qana` namespace. I'm just curious what's the right code design rule here.

Reply:
Oh, I though I *had* put in in the anon namespace...  Will fix, thx